.github/workflows/build-test.yml

```diff
@@ -1,7 +1,9 @@
 name: Build and Test
 
 on:
-  push:
```

Comment:
I would keep the push personally, it's very standard on GitHub.

Reply:
Yes ok, I agree. Just unfortunate that it will run the action upon creating of the repo then. Not a blocking issue though